libraries/resource_firewall_rule.rb

```diff
@@ -18,7 +18,12 @@ class Resource::FirewallRule < Chef::Resource::LWRPBase
                            !!(p.to_s =~ /(udp|tcp|icmp|icmpv6|ipv6-icmp|none)/ || (p.to_s =~ /^\d+$/ && p.between?(0, 142)))
                          end })
     attribute(:direction, kind_of: Symbol, equal_to: [:in, :out, :pre, :post], default: :in)
-    attribute(:logging, kind_of: Symbol, equal_to: [:connections, :packets])
+
+    if Chef::Platform.windows?
```

Comment:
If I did not understand what you meant, you cannot that easily use the same rule for Linux and Windows. But this can be doable.. I will work on it.